leejinwoo/src/components/Seats.tsx

```diff
@@ -0,0 +1,398 @@
+import { useRouter } from "next/router";
+import { ChangeEvent, useEffect, useState } from "react";
+import styled from "styled-components";
+
+const Seats = () => {
+    //인원수
+    const [people, setPeople] = useState<number>(1);
+    // 가격 종류
+    const [price, setPrice] = useState<number>(14000);
+    // 영화선택
+    const [movie, setMovie] = useState<number>(1);
+    // 영화 이름
+    const [movieName, setMovieName] = useState<string>('올빼미');
+    const [result, setResult] = useState<number>(14000);
+    // 라우터쓸거임ㅇㅇ
+    const router = useRouter();
+
+    //영화가 바뀌면 영화 이름을 바꾸게함
+    const onSelectChange = (e:ChangeEvent<HTMLSelectElement>) =>{
+        setMovie(Number(e.target.value));
+
+        if(e.target.value == '1'){
+            setMovieName(String('올빼미'));
+        }
```

Comment:
```suggestion
        if(e.target.value == '1'){
            setMovieName('올빼미');
        }
```

이렇게 해도 충분히 동작할 텐데 String 형을 String 형으로 변환하신 이유가 무엇인가요?